config/routes.rb

```diff
@@ -4,4 +4,11 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   root 'components#index'
+  # All the Api endpoints must be under /api/v1 and must have an extension .json.
+  namespace :api do
+    namespace :v1 do
+      post '/signup', to: 'users#create' # /api/v1/signup.json
```

Comment:
I think it may be better to keep the standard Rails routing concepts here. Use `resources :users, only: :create` -> POST /api/v1/users

Reply:
Alright,

Reply:
Done